grammar.js

```diff
@@ -32,19 +32,23 @@ module.exports = grammar({
   externals: $ => [
     $._automatic_semicolon,
     $._indent,
+    $._outdent,
+    $._simple_string_start,
+    $._simple_string_middle,
+    $._simple_multiline_string_start,
     $._interpolated_string_middle,
-    $._interpolated_string_end,
     $._interpolated_multiline_string_middle,
-    $._interpolated_multiline_string_end,
-    $._outdent,
-    $._simple_multiline_string,
-    $._simple_string,
+    $._raw_string_middle,
+    $._raw_multiline_string_middle,
+    $._single_line_string_end,
+    $._multiline_string_end,
     "else",
     "catch",
     "finally",
     "extends",
     "derives",
     "with",
+    $.error_sentinel,
```

Comment:
The docs suggest having an `error_sentinel` token type. We do not use it, but I found it helpful in debugging.